compiler/z/codegen/OMRLinkage.hpp

```diff
@@ -89,12 +76,6 @@ enum TR_S390LinkageConventions
    TR_JavaPrivate          = 0x1,   ///< Java private linkage
    TR_JavaHelper           = 0x2,   ///< Java helper linkage
```

Comment:
This `enum TR_S390LinkageConventions` shouldn't actually exist. We need to fold this enum into `TR_LinkageConventions` which is an extensible enum exactly for this purpose. I've opened up #2911 to track this.